tests/app/views/test_marketplace.py

```diff
@@ -377,7 +377,7 @@ def test_dos_brief_has_different_link_text_for_logged_in_supplier(self):
 
         assert_equal(qa_link_text.strip(), "Ask a question")
 
-    def test_can_apply_to_live_brief_with_old_supplier_flow(self):
+    def test_can_apply_to_live_brief_with_legacy_supplier_flow(self):
         self.app.config['FEATURE_FLAGS_NEW_SUPPLIER_FLOW'] = False
```

Comment:
Should this test exist twice? Once with the feature flag false and once with the brief published at date before the supplier feature flag?